internetarchive/openlibrary
# Removed duplicate revert button

Revert button styling on Edition pages shows link which should be hidden
Closes #2282

Fix


### Technical
Is a quick fix to the duplicate revert buttons bugs.

### Testing
1.  docker compose up
2. http://localhost:8080/
3. Go find a book
4.  Make Edit on book
5.  Click History
6. Click previous version 
7. Look at side to see revert button

### Screenshot
![Before](https://github.com/internetarchive/openlibrary/assets/56459277/c5ab7d2f-8191-4993-9cb1-926ced76725e)
![After Fix](https://github.com/internetarchive/openlibrary/assets/56459277/066ff151-6e90-48cf-a590-18dbc16f3f96)

### Stakeholders
@jdlrobson
